invoice.go

```diff
@@ -152,26 +191,28 @@ type InvoiceTransferDataParams struct {
 // For more details see https://stripe.com/docs/api#create_invoice, https://stripe.com/docs/api#update_invoice.
 type InvoiceParams struct {
 	Params               `form:"*"`
-	AccountTaxIDs        []*string                     `form:"account_tax_ids"`
-	AutoAdvance          *bool                         `form:"auto_advance"`
-	ApplicationFeeAmount *int64                        `form:"application_fee_amount"`
-	CollectionMethod     *string                       `form:"collection_method"`
-	CustomFields         []*InvoiceCustomFieldParams   `form:"custom_fields"`
-	Customer             *string                       `form:"customer"`
-	DaysUntilDue         *int64                        `form:"days_until_due"`
-	DefaultPaymentMethod *string                       `form:"default_payment_method"`
-	DefaultSource        *string                       `form:"default_source"`
-	DefaultTaxRates      []*string                     `form:"default_tax_rates"`
-	Description          *string                       `form:"description"`
-	Discounts            []*InvoiceDiscountParams      `form:"discounts"`
-	DueDate              *int64                        `form:"due_date"`
-	Footer               *string                       `form:"footer"`
-	OnBehalfOf           *string                       `form:"on_behalf_of"`
-	Paid                 *bool                         `form:"paid"`
-	PaymentSettings      *InvoicePaymentSettingsParams `form:"payment_settings"`
-	StatementDescriptor  *string                       `form:"statement_descriptor"`
-	Subscription         *string                       `form:"subscription"`
-	TransferData         *InvoiceTransferDataParams    `form:"transfer_data"`
+	AccountTaxIDs        []*string                             `form:"account_tax_ids"`
+	AutoAdvance          *bool                                 `form:"auto_advance"`
+	ApplicationFeeAmount *int64                                `form:"application_fee_amount"`
+	AutomaticTax         *InvoiceAutomaticTaxParams            `form:"automatic_tax"`
+	CustomerDetails      *InvoiceUpcomingCustomerDetailsParams `form:"customer_details"`
```

Comment:
This is only for Upcoming and should be lower in the class with "The are all for exclusive use by GetNext"